sklearn/linear_model/glm.py

```diff
@@ -0,0 +1,872 @@
+"""
+Generalized Linear Models with Exponential Dispersion Family
+"""
+
+# Author: Christian Lorentzen <lorentzen.ch@googlemail.ch>
+# License: BSD 3 clause
+
+# TODO: Which name? GeneralizedLinearModel vs GeneralizedLinearRegression.
+#       So far, it is GeneralizedLinearModel, since it could very easily
+#       extended by Bernoulli/Binomial distribution.
```

Comment:
Classifiers have a different API, so best not mix them. Perhaps `GeneralizedLinearRegressor`?